aten/src/THC/THCNumerics.cuh

```diff
@@ -659,6 +666,7 @@ template <>
 struct THCNumerics<float> {
   static inline __host__ __device__ float min() { return -FLT_MAX; }
   static inline __host__ __device__ float max() { return FLT_MAX; }
+  static inline __host__ __device__ float inf() { return 0x7f800000; }
```

Comment:
I am skeptical this actually works. Usually this just would give you the floating point number equivalent to 0x7f800000

Reply:
(And not the bitwise representation. You have to do more work to do that cast.)